cli/src/cli.rs

```diff
@@ -90,118 +74,3 @@ pub struct FactoryCmd {
 	#[structopt(flatten)]
 	pub import_params: ImportParams,
 }
-
-/// Parse command line arguments into service configuration.
-pub fn run<I, T, E>(args: I, exit: E, version: sc_cli::VersionInfo) -> error::Result<()>
-where
-	I: IntoIterator<Item = T>,
-	T: Into<std::ffi::OsString> + Clone,
-	E: IntoExit,
-{
-	type Config<A, B> = Configuration<(), A, B>;
-
-	match parse_and_prepare::<CustomSubcommands, NoCustom, _>(&version, "cennznet-node", args) {
-		ParseAndPrepare::Run(cmd) => cmd.run(
-			load_spec,
-			exit,
-			|exit, _cli_args, _custom_args, mut config: Config<_, _>| {
-				info!("{}", version.name);
-				info!("  version {}", config.full_version());
-				info!("  by Centrality Investments UNlimited, 2018-2020");
```

Comment:
We'll want to re-instate these log messages at some point, not sure where they are handled now

Reply:
They now seem to be coming from Plug: https://github.com/plugblockchain/plug-blockchain/blob/b7e29ddaab5a2bbf08a730424ec5dffcfcd6467f/client/cli/src/commands/runcmd.rs#L458

Reply:
Are you happy with what @cowboy-bebug is suggesting that we re-instate the log in a different task?